### PR TITLE
Migrate existing site v9 patch

### PR DESCRIPTION
**Pull request by jonat123**
Updating the article for migrating an existing site to v9 on cloud.

### CODE REVIEW

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index-v8.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
+---
+versionFrom: 8.0.0
+needsV9Update: "true"
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index-v8.md:3]: Is "needsV9Update" still required? I see an index.md file for v9...

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-versionFrom: 8.0.0
+versionFrom: 9.0.0
 needsV9Update: "true"
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:3]: Is this still required?

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 3]
```diff
@@ -153,44 +110,33 @@ If you on your existing site have been working with members and made changes to
...
-
-1. Rename the old data types on your existing site or
-2. Follow the steps in the [Colliding Data types](../../Troubleshooting/Deployments/Colliding-Datatypes) article
-:::
-
 ## 5. Clone down the Cloud project
 
 With your Umbraco Cloud project ready for the migration, it is time to clone down the project to your local machine.
 
-<iframe width="800" height="450" src="https://www.youtube.com/embed/1uUadErk9vQ?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:172]: Maybe add a link to the "Cloning down your umbraco cloud project" video for reference? https://www.youtube.com/watch?v=e3spd6Nqrf8

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 4]
```diff
@@ -216,18 +162,18 @@ In this next part, it is time to generate the so called UDA-files for all your p
 
 For more details about UDA files, read the [UDA Files](../../Set-Up/Power-Tools/Generating-UDA-files/#what-are-uda-files) article.
 
-* Make sure the folder `/data/revision` on your Umbraco Cloud project is empty
+* Make sure the folder `/umbraco/Deploy/revision` on your Umbraco Cloud project is empty
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:165]: umbraco/Deploy/Revision (R is capital)

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 5]
```diff
@@ -216,18 +162,18 @@ In this next part, it is time to generate the so called UDA-files for all your p
...
   * If you have any files in the folder, you can safely remove those at this point
-* Open a command prompt
-* Navigate to the `/data` folder in your local Umbraco Cloud project files
-* Add an *export* marker by typing `echo > deploy-export`
+* Start your local cloud project
+* Navigate to the settings tab in your project
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:168]: **Settings** section

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 6]
```diff
@@ -216,18 +162,18 @@ In this next part, it is time to generate the so called UDA-files for all your p
...
+* Go to the Deploy tab in the settings section and run the `Extract Schema To Data Files` Command
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:169]: Suggestion: Split the sentence and bold the section/tab drop-down names

Go to the **Deploy** tab in the **Settings** section. Select `Extract Schema To Data Files` from the **Deploy Operations** drop-down and click **Trigger Operation**.

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 7]
```diff
@@ -216,18 +162,18 @@ In this next part, it is time to generate the so called UDA-files for all your p
...
   * Generating the UDA files may take a while, depending on the extent of your project
-  * You will see a `deploy-complete` marker once the export is done
-  * Run `echo > deploy` to check that the UDA files have been generated correctly
-  * When you see a `deploy-complete` marker, it means everything is working as expected
-* You should now see that your `/data/revision` folder has been populated with UDA files corresponding to your projects metadata
+  * You will see a `Last deployment operation completed` status once the export is done
+  * Run `Schema Deployment From Data Files` to check that the UDA files have been generated correctly
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:172]: Suggestion: Rephrase as
Select `Schema Deployment From Data Files` from the **Deploy Operations** drop-down and click **Trigger Operation** to check that the UDA files have been generated correctly

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 8]
```diff
@@ -86,25 +57,16 @@ After making sure that your existing site meets all the requirements for being m
 
 ### Upgrade to latest Umbraco version
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:58]: Upgrade to the latest Umbraco version

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 9]
```diff
@@ -86,25 +57,16 @@ After making sure that your existing site meets all the requirements for being m
...
 
-First order of business is to **upgrade your own Umbraco site to the latest minor version of Umbraco 8**. Why? Because Umbraco Cloud always runs the latest version and you need to make sure your project runs the same Umbraco version as Umbraco Cloud.
+First order of business is to **upgrade your own Umbraco site to the latest minor version of Umbraco 9**. Why? Because Umbraco Cloud always runs the latest version and you need to make sure your project runs the same Umbraco version as Umbraco Cloud.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:60]: The first order

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 10]
```diff
@@ -86,25 +57,16 @@ After making sure that your existing site meets all the requirements for being m
...
 If you need help upgrading your project, we have some excellent [Upgrade instructions](https://our.umbraco.com/documentation/Getting-Started/Setup/Upgrading/general) you can follow. Be thorough when upgrading, as the latest upgrade might contain breaking changes and/or updated configuration.
 
-:::note
-### Umbraco v7
-As of 25th of June 2021 it's no longer possible to migrate v7 projects to Umbraco Cloud since the creation of them has been disabled.
-
-If you wish to host your v7 project on Umbraco Cloud then you'll need to upgrade it to v8 first.
-:::
-
 If you have been using Umbraco Forms on your own project, you will also need to upgrade this to the latest version. You can find and download the latest version of Umbraco Forms under [Projects on Our](https://our.umbraco.com/projects/developer-tools/umbraco-forms/). As with Umbraco CMS we also have documentation on how to [Upgrade Umbraco Forms](https://our.umbraco.com/documentation/Add-ons/UmbracoForms/Installation/ManualUpgrade).
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:66]: As with Umbraco CMS,

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 11]
```diff
@@ -114,14 +76,9 @@ While the site is running you need to:
 * Go the backoffice of your project
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:76]: Go to the backoffice of your project

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 12]
```diff
@@ -153,44 +110,33 @@ If you on your existing site have been working with members and made changes to
...
-
-You can either
-
-1. Rename the old data types on your existing site or
-2. Follow the steps in the [Colliding Data types](../../Troubleshooting/Deployments/Colliding-Datatypes) article
-:::
-
 ## 5. Clone down the Cloud project
 
 With your Umbraco Cloud project ready for the migration, it is time to clone down the project to your local machine.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:115]: With your Umbraco Cloud project ready for migration,

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 13]
```diff
@@ -153,44 +110,33 @@ If you on your existing site have been working with members and made changes to
...
 
-<iframe width="800" height="450" src="https://www.youtube.com/embed/1uUadErk9vQ?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
-
 Run the site locally and verify your own project and the cloned Umbraco Cloud project are using the same Umbraco version. After you've verified this, shut down the site, and it's now time to start merging the two projects.
 
 ## 6. Move and merge files
 
 Merging your existing site into the Umbraco Cloud project is a matter of moving and merging files between the two projects. When following the steps outlined below it is important that you do not overwrite any settings and configuration related to Umbraco Deploy.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:121]: Maybe add this in an important tag as a note, tip

When following the steps outlined below it is important that you do not overwrite any settings and **configurations** related to Umbraco Deploy.

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 14]
```diff
@@ -153,44 +110,33 @@ If you on your existing site have been working with members and made changes to
...
+1. Copy and replace the following folders from your own project to the Umbraco Cloud project
+
+    * `/wwwroot`
+    * `/Views`
+    * `/umbraco`
+
+    * If your existing site uses Umbraco Forms, make sure you **do not overwrite** the `\umbraco\Licenses\umbracoForms.lic` file
+2. Merge Appsettings.JSON files from your existing site with the cloud site.
+
+    * Make sure that you merge your Appsetting.JSON files with the one from your existing site so that the settings you used are moved over to your Cloud project
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:132]: ones

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 15]
```diff
@@ -216,18 +162,18 @@ In this next part, it is time to generate the so called UDA-files for all your p
...
+* Go to the Deploy tab in the settings section and run the `Extract Schema To Data Files` Command
   * Generating the UDA files may take a while, depending on the extent of your project
-  * You will see a `deploy-complete` marker once the export is done
-  * Run `echo > deploy` to check that the UDA files have been generated correctly
-  * When you see a `deploy-complete` marker, it means everything is working as expected
-* You should now see that your `/data/revision` folder has been populated with UDA files corresponding to your projects metadata
+  * You will see a `Last deployment operation completed` status once the export is done
+  * Run `Schema Deployment From Data Files` to check that the UDA files have been generated correctly
+  * When you see a `Last deployment operation completed` status, it means everything is working as expected
+* You should now see that your `/umbraco/Deploy/revision` folder has been populated with UDA files corresponding to your projects metadata
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:174]: project’s

**Review by eshanrnh — COMMENTED** [review]
Looks good 💪Just a couple of suggestions to the grammar.

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index-v8.md (hunk 1 above)
Comment (jonat123) [reply Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index-v8.md:3]: No you are right, i'll remove it :D

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md (hunk 3 above)
Comment (jonat123) [reply Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:172]: Good idea :D

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md (hunk 6 above)
Comment (jonat123) [reply Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:169]: Makes sense, thanks!

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md (hunk 7 above)
Comment (jonat123) [reply Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:172]: Thanks, it makes sense to me :D

**Pull request merged**